applications/tari_merge_mining_proxy/src/main.rs

```diff
@@ -392,14 +444,15 @@ fn handle_connection(
                                             .clone()
                                             .ok_or_else(|| MmProxyError::MissingDataError("Pow data".to_string()))?;
                                         powdata.pow_data = bincode::serialize(&pow_data).map_err(|_| {
-                                            MmProxyError::ParseError(format!("Failure to serialize block "))
+                                            MmProxyError::ParseError("Failure to serialize block".to_string())
                                         })?;
                                         tariheader.pow = Some(powdata);
                                         block.header = Some(tariheader);
                                         trace!(target: LOG_TARGET, "Tari Block {:?}", block);
                                         match rt.block_on(grpcclient.submit_block(block)) {
                                             Ok(result) => {
                                                 result.into_inner();
+                                                transient.tari_prev_submit_height = transient.tari_height;
                                             },
                                             Err(e) => {
                                                 debug!(target: LOG_TARGET, "{:#}", e);
```

Comment:
```suggestion
                                                error!(target: LOG_TARGET, "{:#}", e);
```

Reply:
might be warn, I am not sure which result we are looking there at, the result of the grpc submit, or die actual submit block call. 

If its the grpc fail its error. 
If its the actual call, its a warn